util/build_configs/cray-internal/setenv-xc-aarch64.bash

```diff
@@ -285,7 +285,7 @@ else
         load_module $target_prgenv
         load_module_version $target_compiler $target_version
 
-        # pin to versions of mpich/libsci that work with gen_version_gcc
+        # pin to versions of mpich/libsci that work with target_version_gcc
         load_module_version cray-mpich 7.7.13
         load_module_version cray-libsci 19.06.1
```

Comment:
Should be able to remove these now (only needed them since we were pinned to an old version of gcc)